fhir-bulkimportexport/src/main/java/com/ibm/watson/health/fhir/bulkexport/BulkExportBatchLet.java

```diff
@@ -103,23 +109,28 @@
     @Inject
     @BatchProperty(name = "cos.credential.ibm")
     String cosCredentialIbm;
-    
-    
+
     /**
      * The Cos object name.
      */
     @Inject
     @BatchProperty(name = "cos.bucket.objectname")
     String cosBucketObjectName;
-    
+
+    /**
+     * The Cos object name.
+     */
+    @Inject
+    @BatchProperty(name = "cos.bucket.maxfilesize")
+    String cosBucketMaxFileSize;
```

Comment:
mark as private? same comment in the class files (there is a bit of a side-effect)